hydra_agent/hydra_graph.py

```diff
@@ -21,12 +20,12 @@ def get_apistructure(self,entrypoint_node, api_doc):
         for support_property in api_doc.entrypoint.entrypoint.supportedProperty:
             if isinstance(
                     support_property,
-                    hydrus.hydraspec.doc_writer.EntryPointClass):
+                    doc_writer.EntryPointClass):
```

Comment:
@de-sh There is an error `doc_writer is not defined`. Fix this error.

Reply:
Its not being imported at the top.

Reply:
yes @ShravanDoda, you are right.  @de-sh  import it.

Reply:
Done :+1: